cmake/compilers/GNU.cmake

```diff
@@ -105,6 +105,11 @@ endif ()
 set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -fno-strict-overflow -fno-delete-null-pointer-checks -fwrapv)
 set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -Wformat -Wformat-security -Werror=format-security
     -fstack-protector-strong )
+if (NOT CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
+    set(TBB_LIB_COMPILE_FLAGS ${TBB_LIB_COMPILE_FLAGS} $<$<NOT:$<VERSION_LESS:${CMAKE_CXX_COMPILER_VERSION},8.0>>:-fcf-protection=full>)
+endif ()
```

Comment:
> * -fcf-protection=full flags - not supported in Emscripten and on aarch64.

Isn't a no-Emscripten check needed?

Reply:
As I understand em++ is LLVM-Clang-based compiler, so GCC is not used for Emscripten. Let me know if it's not the case.

Reply:
You are right, but it seems it that it can be mixed with GCC somehow, as mentioned in #1391. Or was that just extra carefulness? @isaevil, do you know the details?

Reply:
Unfortunately, I don't know particular details about this GNU+Emscripten scenario. But as an extra carefulness, as @dnmokhov said, I would prefer to add the check.

Reply:
Added the check.